app/src/main/java/com/github/epfl/meili/models/User.kt

```diff
@@ -9,6 +9,7 @@ import kotlinx.parcelize.Parcelize
 data class User(
     val uid: String,
     val username: String,
+    val email: String
 ) : Parcelable {
```

Comment:
Just a question about the choice here, why did you choose to use Parcelable to implement User?

Reply:
Using Parcelable allows us to save it and retrieve it from FirebaseDatabase more easily